config/nginx.conf.erb

```diff
@@ -52,12 +52,12 @@ http {
     <% if ENV["API_URL"] %>
       location <%= ENV["API_PREFIX_PATH"] || "/api/" %> {
         proxy_pass <%= ENV["API_URL"] %>;
-        proxy_set_header X-Real-IP $remote_addr;
```

Comment:
http://nginx.org/en/docs/http/ngx_http_proxy_module.html#example -- this is straight from the Nginx docs.